Parsedown.php

```diff
@@ -1453,8 +1458,19 @@ protected function elements(array $Elements)
         return $markup;
     }
 
+    #
+    # - [x] AND - [ ]
+    protected function checkbox(&$text,&$attributes)
+    {
+        if(strpos($text,'[x]')!==false||strpos($text,'[ ]')!==false){
+            $attributes = array("style"=>"list-style: none;");
+            $text = str_replace(array('[x]','[ ]'), array(
+                '<input type="checkbox" checked="true" disabled="true">',
```

Comment:
This should be refactored into the internal way of representing an element so that it doesn't break when #495 is merged (see #507 for an example).